dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/AbstractMethodProblem.java

```diff
@@ -22,7 +22,23 @@
  * as {@link AbstractMethodError}s at runtime.
  */
 final class AbstractMethodProblem extends IncompatibleLinkageProblem {
+  private final MethodSymbol methodSymbol;
+
   AbstractMethodProblem(ClassFile sourceClass, ClassFile targetClass, MethodSymbol methodSymbol) {
-    super("is not implemented in the class", sourceClass, targetClass, methodSymbol);
+    this(sourceClass, targetClass, methodSymbol, null);
+  }
+
+  private AbstractMethodProblem(
+      ClassFile sourceClass,
+      ClassFile targetClass,
+      MethodSymbol methodSymbol,
+      LinkageProblemCause cause) {
```

Comment:
This shouldn't even be passed here if we don't know it when the object is constructed.